src/main/resources/db/changesets/20221014_duosfirecloudgroup.yaml

```diff
@@ -0,0 +1,45 @@
+databaseChangeLog:
+  - changeSet:
+      id: duosfirecloudgroup
+      author: okotsopo
+      changes:
+        - createTable:
+            tableName: duos_firecloud_group
+            columns:
+              - column:
+                  name: id
+                  type: ${uuid_type}
+                  defaultValueComputed: ${uuid_function}
+                  constraints:
+                    primaryKey: true
```

Comment:
Whoops, this new `id` column should not be a primary key… otherwise we will permit the creation of records with the same DUOS ID. I will update and expand unit tests.